src/core.js

```diff
@@ -746,7 +746,7 @@ define([
             return false;
         });
 
-        $modalBody.find('#findUsagesSearch').on('keyup inserted.atwho', function () {
+        $modalBody.find('#findUsagesSearch').on('keyup keypress inserted.atwho', function (e) {
```

Comment:
Gah. I was on the wrong branch and accidentally pushed `debounce` to master https://github.com/dimagi/Vellum/commit/65cccf5d3ccc676cedc05b8d89247b3e3293fb68 I'm not going to worry about reverting it right now as it's a small change, but that should also be reviewed